app/src/main/java/com/codesoom/assignment/domain/CatToy.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.domain;
+
+public class CatToy extends Product{
```

Comment:
굳이 `CatToy` 타입을 선언할 필요가 있을까요? 프로그래밍은 추상적인 개념을 다룰수록 확장성을 더 많이 확보할 수 있게 되는 활동입니다. `CatToy` 타입을 선언하게 되면, `DogToy`, `CatFood`, `CatBall`, `CatToilet`... 계속해서 타입을 만들어야 할 겁니다. `class`라는 키워드의 의미를 다시 한번 생각해 봅시다.

```suggestion
public class Product {
```

그리고 상속은 좋지 않습니다. 상속이 좋지 않은 이유에 대해서도 조사해 보세요.

Reply:
상속이 좋지 않은 이유에 대해서 조사해 보았습니다.

1. 부모의 캡슐화를 보장하지 않는다. 
- 부모 객체가 대체할 수 있는 is-a 관계가 아닐 경우, 자식 객체에서 부모의 public 메소드를 사용해 원치 않는 동작을 할 수 있다.

2. 결합도가 높아진다.
- 상속을 이용하면 결합도가 높아져 부모 객체의 메소드가 변경되면, 이를 상속한 자식 객체들의 메소드가 변경되는 경우가 많아진다.

Reply:
추상적인 이름인 Product를 사용하는 게 더 좋겠네요.